rs/nns/governance/api/src/bitcoin.rs

```diff
@@ -0,0 +1,28 @@
+use std::str::FromStr;
+
+#[derive(candid::CandidType, serde::Serialize, candid::Deserialize, Clone, Debug, Copy)]
+pub enum BitcoinNetwork {
```

Comment:
I'm not seeing the reason that it belongs here. IIUC the only usage of this is for NNS Governance to decode, modify and encode the payload for proposal execution. I think conceptually the type should be internal to NNS Governance.

Reply:
Registry uses this and Governance uses this.  Where do the proposals come from though?

I'm wondering - what's a better place for it?  I wonder if there's somewhere in the bitcoin canister it ought to go.